examples/undocumented/python_modular/graphical/interactive_clustering_demo.py

```diff
@@ -11,10 +11,9 @@
 from PyQt4.QtGui import *
 
 import matplotlib
-from matplotlib import mpl
 from matplotlib.colorbar import make_axes, Colorbar
 from matplotlib.backends.backend_qt4agg import FigureCanvasQTAgg as FigureCanvas
-from matplotlib.backends.backend_qt4agg import NavigationToolbar2QTAgg as NavigationToolbar
```

Comment:
`NavigationToolbar2QTAgg` replaced by `NavigationToolbar2QT` as the former gives import error.